src/xutils.hpp

```diff
@@ -0,0 +1,36 @@
+/***************************************************************************
+* Copyright (c) 2018, Martin Renou, Johan Mabille, Sylvain Corlay and      *
+* Wolf Vollprecht                                                          *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+
+#ifndef XPYT_UTILS_HPP
+#define XPYT_UTILS_HPP
+
+#include <vector>
+
+#include "xeus/xjson.hpp"
+#include "xeus/xcomm.hpp"
+
+#include "pybind11/pybind11.h"
+
+namespace py = pybind11;
+
+namespace xpyt
+{
+
+    xeus::xjson pyobj_to_xjson(py::object obj);
+    py::object xjson_to_pyobj(const xeus::xjson& json);
```

Comment:
Nitpicking: how about

`json_to_py`
`py_to_json`

Reply:
Also, how about inluding `nlohmann/json.hpp` and dropping xjson wherever possible?

@JohanMabille

Reply:
If I do that, do you think I should define `xpyt::xjson`?

Reply:
I don't think so. The xjson header of xeus comes from our previous usage of rapidjson. We were adding utility functions.